elements/header.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
```

Comment:
somehow this broke the hamburger menu, it's not opening up any more for me

Reply:
Hm, I will take a look when I have time

Reply:
fixed by @martinheidegger in this commit https://github.com/datproject/dat-desktop/pull/450/commits/04697c070fbcc93a836187807e6634d0e0e8cfd2 yay!